physics/forkable.hpp

```diff
@@ -84,9 +84,10 @@ class ForkableIterator {
   void CheckNormalizedIfEnd();
 
   // |ancestry_| is never empty.  |current_| is an iterator in the timeline
-  // for |ancestry_.front()|.  |current_| may be at end.
+  // for |ancestry_.back()|.  |current_| may be at end.
   TimelineConstIterator current_;
-  std::deque<not_null<Tr4jectory const*>> ancestry_;  // Pointers not owned.
+  absl::InlinedVector<not_null<Tr4jectory const*>, 2>
+      ancestry_;  // Pointers not owned.
```

Comment:
Remove the comment, it's a relic of the distant past.

Reply:
Done.